### ISSUE TITLE
Environment Variables

### ISSUE THREAD

**Issue by jeff363b** [MaarifaMaarifa/series-troxide#191]
How do I change my Client ID and Client Secret in the Trakt setup?  What I have in there is wrong and I can't find how to fix it.  I even tried to redeploy the stack to no avail.

Jeff

**Comment by MaarifaMaarifa** [comment]
The program has got no feature to modify Client Secret and Client ID environment variables. You'll have to change them first (i.e using your shell) before launching the program.

**Comment by jeff363b** [comment]
Thank you. How do I get into the shell? Or can it be changed in portainer?

--
Jeff

From: Maarifa Maarifa ***@***.***> ***@***.***)
Reply: MaarifaMaarifa/series-troxide ***@***.***> ***@***.***)
Date: February 26, 2024 at 4:15:00 AM
To: MaarifaMaarifa/series-troxide ***@***.***> ***@***.***)
CC: jeff363b ***@***.***> ***@***.***), Author ***@***.***> ***@***.***)
Subject: Re: [MaarifaMaarifa/series-troxide] Environment Variables (Issue #191)

**Comment by MaarifaMaarifa** [comment]
Yes you can configure it in portainer. You can just add the environment variables in the environment section in the docker compose file.

**Comment by jeff363b** [comment]
Can you give me an example of how it would be added…somewhat a novice on the portainer…

--
Jeff

From: Maarifa Maarifa ***@***.***> ***@***.***)
Reply: MaarifaMaarifa/series-troxide ***@***.***> ***@***.***)
Date: February 26, 2024 at 7:32:32 PM
To: MaarifaMaarifa/series-troxide ***@***.***> ***@***.***)
CC: jeff363b ***@***.***> ***@***.***), Author ***@***.***> ***@***.***)
Subject: Re: [MaarifaMaarifa/series-troxide] Environment Variables (Issue #191)

**Comment by Hoodah** [comment]
I've added the environment variables to my docker instance as you can see here:
![image](https://github.com/MaarifaMaarifa/series-troxide/assets/10746038/87357f2a-27cb-4747-8fbd-ca88889c667e)
However Series-Troxide reports:
![image](https://github.com/MaarifaMaarifa/series-troxide/assets/10746038/b117a444-426a-4caa-ad7a-9eccbf40b81a)

**Comment by MaarifaMaarifa** [comment]
> I've added the environment variables to my docker instance as you can see here: ![image](https://private-user-images.githubusercontent.com/10746038/316609079-87357f2a-27cb-4747-8fbd-ca88889c667e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwNzU3NDEsIm5iZiI6MTcxMjA3NTQ0MSwicGF0aCI6Ii8xMDc0NjAzOC8zMTY2MDkwNzktODczNTdmMmEtMjdjYi00NzQ3LThmYmQtY2E4ODg4OWM2NjdlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAyVDE2MzA0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFkZDhmN2I4OGM0NmQ2Mzg5ZmE4N2M2MTI4MTEwMTc3MjQ0YmZiZmZjNTlmZjEwNmNiYzcxOTA2OTc5ZTEyNDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9NspbTuTg6ymyMeqL9FstCb-DtjLWxUSj-bskYxJs1U) However Series-Troxide reports: ![image](https://private-user-images.githubusercontent.com/10746038/316609406-b117a444-426a-4caa-ad7a-9eccbf40b81a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwNzU3NDEsIm5iZiI6MTcxMjA3NTQ0MSwicGF0aCI6Ii8xMDc0NjAzOC8zMTY2MDk0MDYtYjExN2E0NDQtNDI2YS00Y2FhLWFkN2EtOWVjY2JmNDBiODFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAyVDE2MzA0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxMGM1ZGRlMjJiM2U3YTE0Yjc0ZDk3YzQyODY5YWUzOGI1MzA2Njc3NDU3YzQyM2M5NWRkZGI0ODE1MmEyMDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Xd4drXQT_6g1RrZg_QJcsAcrKZE3jIS_VxtNiwcW1jE)

You can check out [docker-series-troxide](https://github.com/linuxserver/docker-series-troxide)'s readme as it may provide a better explanation on how to set up environment variables on a dockerized series troxide.

**Comment by Hoodah** [comment]
I have successfully added the environment variables, as shown in the screenshot, the issue is that series-troxide isn't reading them, and there are no errors in the log.
